library/Icinga/Cli/Screen.php

```diff
@@ -94,7 +94,7 @@ public function colorize($text, $fgColor = null, $bgColor = null)
     public static function instance()
     {
         if (self::$instance === null) {
-            if (function_exists('posix_isatty') && posix_isatty(STDOUT)) {
+            if (function_exists('posix_isatty') && posix_isatty(STDERR)) {
```

Comment:
This breaks colored output if stderr is being redirected but stdout is not. Please make `Screen` configurable so that the log output that uses colors defines that it wants to use stderr. stdout must be the default.